app/models/financialDetails/DocumentDetail.scala

```diff
@@ -42,6 +42,9 @@ case class DocumentDetail(taxYear: String,
                           amountCodedOut: Option[BigDecimal] = None
                          ) {
 
+  val interestIsPartPaid: Boolean = interestOutstandingAmount.getOrElse[BigDecimal](0) != latePaymentInterestAmount.getOrElse[BigDecimal](0)
```

Comment:
* are you sure this change is expected to be part of this story?
* this would be good to have basic unit test around this method before changing it, but pls check first if this is really required change;